mono/metadata/marshal.c

```diff
@@ -990,7 +990,8 @@ mono_string_builder_to_utf8 (MonoStringBuilder *sb)
  *
  * Returns: a utf16 string with the contents of the StringBuilder.
  *
- * The return value must not be freed.
+ * The return value must be released with mono_marshal_free.
```

Comment:
Some code seems to rely on not needing to free, does this need to change too: [marshal.c#L2474-L2475](https://github.com/mono/mono/blob/bea97b9d05fbb675bce541647d77b7c15d41fe03/mono/metadata/marshal.c#L2474-L2475) ?